examples/animation-aframe-logo/index.html

```diff
@@ -8,106 +8,133 @@
   </head>
   <body>
     <a-scene>
+      <a-assets>
+        <a-asset-item id="tree" src="../_models/tree1/tree1.dae"></a-asset-item>
+      </a-assets>
 
       <!-- Quasi-isometric camera -->
       <a-entity>
-        <a-animation attribute="position" from="0 5 0" to="0 0 0" dur="4000" easing="ease-out"></a-animation>
-        <a-camera id="camera" near="1000" far="4000" position="0 880 1290" fov="2.2" cursor-visible="false" rotation="-34 0 0" wasd- controls-enabled="false" look-controls-enabled="false"></a-camera>
+        <a-animation attribute="position" from="0 5 0" to="0 0 0" dur="4000"
+                     easing="ease-out"></a-animation>
+        <a-camera id="camera" near="1000" far="4000" position="0 880 1290" fov="2.2"
```

Comment:
[http://localhost:9000/examples/animation-aframe-logo/](http://localhost:9000/examples/animation-aframe-logo/)

this was already busted on `dev` in both Chrome and Firefox:

![screenshot 2016-02-17 03 22 48](https://cloud.githubusercontent.com/assets/203725/13108079/baf0a2fc-d525-11e5-8716-36bc85204e70.png)
<img width="1363" alt="screenshot 2016-02-17 03 22 32" src="https://cloud.githubusercontent.com/assets/203725/13108080/baf13762-d525-11e5-81e1-aff95db6da5b.png">

and in this branch the Chrome one still is broken, but the Firefox one looks good (nice work):

![screenshot 2016-02-17 03 11 46](https://cloud.githubusercontent.com/assets/203725/13108102/d6c9ad84-d525-11e5-846e-4111b05a13d9.png)
<img width="1363" alt="screenshot 2016-02-17 03 11 44" src="https://cloud.githubusercontent.com/assets/203725/13108103/d6cabfe4-d525-11e5-8640-87020cd1661f.png">